bot/view.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, controller: BotController, service: BotService, auth_controll
         self.controller = controller
         self.service = service
         self.auth_controller = auth_controller
+        self.admin_emails = admin_emails
```

Comment:
What is the purpose of this attribute?

Reply:
Currently, the admin emails are predefined in the environment, so that attribute stores which email addresses are admin emails.

Reply:
what will that admin emails be used for inside this `BotViewV1`?

Reply:
Only admins can see users on the navbar, so we need an admin_emails list.